src/App.module.css

```diff
@@ -42,7 +42,7 @@
 .pageContent{
   display: flex;
   position: relative;
-  margin-top: 5rem;
+  /* margin-top: 5rem; */
```

Comment:
can remove this comment if not reqd

Reply:
Yes kept it for the purpose of other pages setup so would adjust it accordingly